src/extensions.rs

```diff
@@ -65,7 +81,8 @@ impl CommandExt for Command {
         } else {
             Err(CommandError::NonZeroExitCode {
                 status,
-                command: self.command_pretty(),
+                command: self
+                    .command_pretty(verbose, |ref cmd| STRIPPED_BINS.iter().any(|f| f == cmd)),
```

Comment:
Does this work? Don't we provide commands using the path of docker? Also, this should probably also support `cargo` as well. For example, we do `which::which`. We could map the values for the path to the actual unprefixed value?

Reply:
The const includes cargo, so it does it already

it works as in we never display `/usr/bin/cargo` unless we're in verbose

Reply:
> I guess we could just discard the result of which::which, since it has to be in-the-path anyway, or provided as a full path if using CROSS_CONTAINER_ENGINE and not in-the-path.

That makes sense to me, but there could be a point to having the full path.